Migrations.php

```diff
@@ -41,13 +41,21 @@ public function migrate($migrations, Request $request, TargetDb $targetDb)
                 $this->log('Processed ' . $migration->getName());
             }
         } catch (\Exception $e) {
-            $targetDb->rollBack();
+            //Since php8, PDO::inTransaction() now reports the actual transaction state of the connection, rather than an approximation maintained by PDO. If a query that is subject to "implicit commit" is executed, PDO::inTransaction() will subsequently return false, as a transaction is no longer active
+            //inTransaction check fixes warning raised due to implicit commit change
+            if ($targetDb->getDb()->getConnection()->inTransaction()) {
```

Comment:
@AltamashShaikh here we would need to check if the method exists if I see this right. For example if `mysqli` DB adapter is used.

Reply:
Updated the condition, please check